src/raftstore/store/store.rs

```diff
@@ -1745,12 +1745,12 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             let min_index = peer.get_min_progress() + 1;
             let low = cmp::max(min_index, state.get_min_index());
             // TODO: move this into raft module.
-            let entries = if low >= state.get_commit() {
+            let entries = if low >= state.get_commit() + 1 {
```

Comment:
PTAL @foreverbell

Reply:
can you explain why +1 now?

Reply:
Please add some comments that we now include the `PrepareMerge`.